math/mathcore/inc/Fit/FitExecutionPolicy.h

```diff
@@ -1,9 +0,0 @@
-#ifndef ROOT_Fit_FitExecutionPolicy
-#define ROOT_Fit_FitExecutionPolicy
-namespace ROOT{
-   namespace Fit{
-      enum class ExecutionPolicy { kSerial, kMultithread, kMultiprocess };
```

Comment:
Did we not release this already? If so this is a backward incompatible change.

Reply:
Maybe I can keep this file aliasing the execution policies to the new ones and add a deprecation note